CMakeLists.txt

```diff
@@ -50,4 +50,15 @@ target_link_libraries(tfs ${MYSQL_CLIENT_LIBS} ${LUA_LIBRARIES} ${Boost_LIBRARIE
 
 set_target_properties(tfs PROPERTIES COTIRE_CXX_PREFIX_HEADER_INIT "src/otpch.h")
 set_target_properties(tfs PROPERTIES COTIRE_ADD_UNITY_BUILD FALSE)
+
+# clang-format
+find_program(CLANG_FORMAT NAMES clang-format-4.0 clang-format-3.9)
```

Comment:
Why those names? It's named just `clang-format` here.

Reply:
IIRC ubuntu only has clang-format-x.y

Reply:
"clang-format" version is 3.8 which is not compatible with the .clang-format I provided.